docs/install_guides/installing-openvino-raspbian.md

```diff
@@ -26,38 +26,38 @@
 .. tab:: Software Requirements
 
   * CMake 3.7.2 or higher
-  * Python 3.6-3.8, 32-bit
+  * Python 3.6 - 3.9, 32-bit
+     * Note that OpenVINO will stop the support for Python 3.6 shortly. Python 3.7 - 3.9 are recommended.
 
 @endsphinxdirective
 
-## Overview
-
-This guide provides step-by-step instructions on how to install the Intel® Distribution of OpenVINO™ toolkit for Raspbian OS. The following steps will be covered:
-
-1. <a href="#install-openvino">Install the Intel® Distribution of OpenVINO™ Toolkit</a>
-2. <a href="#install-external-dependencies">Install External Software Dependencies</a>
-3. <a href="#set-the-environment-variables">Configure the Environment</a>
-4. <a href="#add-usb-rules">Add USB rules for an Intel® Neural Compute Stick 2 device (Optional)</a>
-6. <a href="#workflow-for-raspberry-pi">Learn About Workflow for Raspberry Pi (Optional)</a>
-
-## <a name="install-openvino"></a>Step 1: Install the Intel® Distribution of OpenVINO™ Toolkit
+## <a name="install-openvino"></a>Step 1: Download and Install OpenVINO Runtime
 
 1. Open the Terminal or your preferred console application.
-2. Go to the directory in which you downloaded the Intel® Distribution of OpenVINO™ toolkit. This document assumes this is your `~/Downloads` directory. If not, replace `~/Downloads` with the directory where the file is located.
+2. Create an installation folder for OpenVINO. If the folder already exists, skip this step.
+   ```sh
+   sudo mkdir -p /opt/intel
+   ```
+   > **NOTE**: The `/opt/intel` path is the recommended folder path for administrators or root users. If you prefer to install OpenVINO in regular userspace, the recommended path is `/home/<USER>/intel`. You may use a different path if desired.
+
+3. Go to your `~/Downloads` directory and download OpenVINO Runtime archive file for Debian from [OpenVINO package repository](https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/).
    ```sh
    cd ~/Downloads/
+   sudo wget https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/l_openvino_toolkit_runtime_rhel8_p_2022.1.0.643.tgz -O openvino_2022.1.0.643.tgz
```

Comment:
why rhel? while we are about documentation for raspbian

Reply:
Thanks for pointing that out. Correction is under way.